buildingmotif/label_parsing/combinators.py

```diff
@@ -118,18 +117,19 @@ def __call__(self, target: str) -> List[TokenResult]:
 class abbreviations(Parser):
     """Constructs a choice combinator of string matching based on a dictionary."""
 
-    def __init__(self, patterns: dict):
-        parsers = [string(s, Constant(URIRef(t))) for s, t in patterns.items()]
+    def __init__(self, patterns):
+        patterns = patterns
+        parsers = [string(s, Constant(t)) for s, t in patterns.items()]
         self.choice = choice(*parsers)
 
-    def __call__(self, target: str):
-        return self.choice(target)
+    def __call__(self, target):
+        return self.choice(target.upper())
```

Comment:
If we remove this `.upper()`, will that break your code? I would rather this kind of normalization happen in a specialization of the `abbreviations` parser (like a subclass) rather than in the root class.